samples/net/wpan_serial/README.rst

```diff
@@ -0,0 +1,40 @@
+.. _wpanusb_serial-sample:
+
+Application implementing 802.15.4 "serial-radio" protocol
+#########################################################
+
+How to use
+**********
+
+#. Build and flash Zephyr application to the board with 802.15.4 radio.
+#. Connect board to Linux PC, /dev/ttyACM[number] should appear.
+#. Run Contiki-based native border router (6lbr, native-router, etc)
+   Example for Contiki:
+
+   .. code-block:: console
+
+     $ cd examples/ipv6/native-border-router
+     $ make
+     $ sudo ./border-router.native -v5 -s ttyACM0 fd01::1/64
+
+Now you have Contiki native border router, simple web server should show some
+802.15.4 devices. Web server address is printed in the border-router output
+screen:
+
+
+.. code-block:: console
+
+  ...
```

Comment:
The content block for the code-block directive needs to be indented (as you did for the previous code-block).

Reply:
This does not need to be intended since this is new block